src/lib.rs

```diff
@@ -568,6 +568,46 @@ impl<'a> Record<'a> {
 }
 
 /// Metadata about a log message.
+///
+/// # Use
+///
+/// `Metadata` structs are created when users of the library use
+/// logging macros.
+///
+/// They are consumed by implementations of the `Log` trait in the
+/// `enabled` method. 
+///
+/// `Record`s use `Metadata` to determine the log message's severity
+/// and target.
+///
+///
```

Comment:
Please remove the extra blank line. Rustdoc treats this just like one blank line anyway.